app/Config/Logger.php

```diff
@@ -38,7 +38,7 @@ class Logger extends BaseConfig
      *
      * @var array|int
      */
-    public $threshold = 4;
+    public $threshold = 5;
```

Comment:
I would like to see some more justification for a base log level. Logs are a type of persistence and connected to a lot of value in enterprise. Personally I'm for more logging (I always set 1,2,3,4,8 even in production).

Reply:
I've reverted for now the change in `$threshold`